cpp/benchmarks/join/generate_input_tables.cuh

```diff
@@ -180,3 +188,93 @@ void generate_input_tables(key_type* const build_tbl,
 
   CUDF_CHECK_CUDA(0);
 }
+
+template <bool Nullable>
+std::pair<std::unique_ptr<cudf::table>, std::unique_ptr<cudf::table>> generate_input_tables(
+  std::vector<cudf::type_id> const& key_types,
+  cudf::size_type build_table_numrows,
+  cudf::size_type probe_table_numrows,
+  int multiplicity,
+  double selectivity,
+  rmm::cuda_stream_view stream)
```

Comment:
How does this new function differ from the existing `generate_input_tables`? Does it add support for generating nullable data? If so, there might be quite a bit of duplicate code that both functions share, and we could probably clean things up by refactoring.